about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/about.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+
+    <title>About Apple Chemicals</title>
+
+</head>
+<body>
+  <header class="d-flex flex-d-column">
+    <!-- top row -->
+    <nav class="topRow">
+      <ul class="nav-topRow d-flex">
+        <li class="disapear"><a href="#"><img src="./img/facebook.svg" alt="facebook Icon"></a></li>
+        <li class="disapear"><a href="#"><img src="./img/twitter.svg" alt="facebook Icon"></a></li>
+        <li class="disapear"><a href="#">English</a></li>
+        <li class="disapear"><a href="#">My Page</a></li>
+        <li class="disapear"><a href="#">Logout</a></li>
+      </ul>
+    </nav>
+    <!-- nav bar -->
+    <nav class="navbar d-flex flex-d-column">
+      <a class="logo d-flex" href="./index.html"><img class="logoImg" src="img/logo.jpg" alt="logoImg"></a>
```

Comment:
- [ ] Please ensure that we do not include the logo for the mobile view in the navigation bar because, as stated in the requirement, we should stick as closely to the original design as possible, and the original design does not include the logo in the navigation bar for the mobile view. :ok_hand: 😊
What I mean is this :point_down: 

### Your design :point_down: 
![Screenshot from 2023-03-15 21-26-58](https://user-images.githubusercontent.com/89228743/225447905-07d50a8f-1f62-479b-af49-3aa88d10b389.png)

### The original design :point_down: 
![Screenshot from 2023-03-15 21-27-37](https://user-images.githubusercontent.com/89228743/225447921-fd722f2f-2717-47d1-941a-db4b1919454a.png)